index.html

```diff
@@ -151,17 +151,15 @@ <h3 class="fw-bold mt-3 subline ms-3">Remodelación</h3>
 			<!-- Sobre nosotros -->
 			<section style="background-color: white;">
 				<div class="p-5">
-					<h2><strong class="fw-bold text-color-main h1">PROFESIONALES</strong> PARA TI</h2>
-					<h2 class="line-h">SOBRE NOSOTROS</h2>
 					<div class="mx-5">
-						<p class="fst-italic mt-5 h5" style="text-indent: 2rem;">Somos Carpitec43 una empresa líder en carpinteria general. Fundada en 1968
-							con una trayectoria de 53 años, encargados de llevarte siempre lo mejor en
-						materiales y diseño, a la vanguardia con los standares y equipos actuales. Ubicada en </p>
-						<blockquote class="mt-5"><p class="fst-italic fw-bold">
+						<h1 style="text-align: left ;"><strong class="fw-bold text-color-main h2">PROFESIONALES</strong> PARA TI </h1>
```

Comment:
No hace falta stilos en línea, https://getbootstrap.com/docs/5.0/utilities/text/ usa las clases de bootstrap 5 para ello, y como nota no uses stilos en línea puesto que es mala práctica
```suggestion
						<h1 class="text-start"><strong class="fw-bold text-color-main h2">PROFESIONALES</strong> PARA TI </h1>
```